pycaret/internal/plots/time_series.py

```diff
@@ -543,16 +514,36 @@ def plot_pacf(
     data_kwargs: Optional[Dict] = None,
     fig_kwargs: Optional[Dict] = None,
 ) -> PlotReturnType:
-    """Plots the PACF on the data provided"""
-    fig, return_data_dict = None, None
-
+    """Plots the PACF on the data provided
+
+    Parameters
+    ----------
+    data : pd.Series
+        Data whose correlation plot needs to be plotted
+    fig_defaults : Dict[str, Any]
+        The defaults dictionary containing keys for "width" and "height" (mandatory)
+    model_name : Optional[str]
+        The model_name must be passed to model residuals, while it should be left None
```

Comment:
Similar to ACF plot, the wording here is confusing. Could you please change it to the following.

> If the correlation plot is for model residuals, then, model_name must be passed for proper display of results. If the correlation plot is for the original data, model_name should be left None (name is derived from the data passed in this case).